data/types/food/food_test.go

```diff
@@ -452,6 +455,11 @@ var _ = Describe("Food", func() {
 				Entry("does not modify the datum; nutrition missing",
 					func(datum *food.Food) { datum.Nutrition = nil },
 				),
+				Entry("does not crash when prescriptor is not set",
+					func(datum *food.Food) {
+						datum.Prescriptor = common.NewPrescriptor()
+					},
+				),
```

Comment:
This test here does nothing.
Put it in `normalizes the datum for rescueCarbs and any other prescriptor; PrescribedNutrition is changed to nil`:
```
				Entry("does not crash when prescriptor is not set",
					func(datum *food.Food) {
						datum.Prescriptor.Prescriptor = nil
					},
				),
```